apps/auth_content/admin.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib import admin
+
+from import_export.admin import ExportMixin
+
+from .models import ClosedPODContactInformation
+
+
+@admin.register(ClosedPODContactInformation)
+class ClosedPODContactInformationAdmin(ExportMixin, admin.ModelAdmin):
+    list_display = (
+        "id",
+        "user",
+        "facility_name",
+        "facility_id",
+    )
+    search_fields = (
+        "user",
```

Comment:
It looks like this currently fails with "Related Field got invalid lookup: icontains". I think we probably want to search users' emails?
```suggestion
        "user__email",
```
Or maybe also `user__first_name` and `user__last_name`?